apis/coursebatchcertificateapi/coursebatchcertificateapi.yaml

```diff
@@ -0,0 +1,455 @@
+      - name: Authorization
+        in: header
+        required: true
+        type: string
+        description: 'To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: 'OK ! Operation successful."Certificate generation" operation was executed successfully.'
+          schema:
+            allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/IssueCertificateResponse'
+        '400':
+          description: 'BAD REQUEST. The "Certificate generation" operation failed. Input for a mandatory parameter may be missing.'
+          schema:
+            $ref: '#/definitions/IssueCertificateErrorResponse'
+        '500':
+          description: 'INTERNAL SERVER ERROR ! Looks like something went wrong. These errors are tracked automatically. Try refreshing the page. If the problem persists, contact info@sunbird.org.'
+definitions:
+  AddCertificateTemplateRequest:
+    title: AddCertificateTemplateRequest
+    type: object
+    properties:
+      request:
+        $ref: '#/definitions/AddCourseBatchTemplateRequest'
+    required:
+    - request
+  AddCourseBatchTemplateRequest:
+    title: AddCourseBatchTemplateRequest
+    type: object
+    properties:
+      batch:
+        $ref: '#/definitions/AddCourseBatchTemplate'
+    required:
+    - batch
+  AddCourseBatchTemplate:
+    title: AddCourseBatchTemplate
+    type: object
+    properties:
+      courseId:
+        description: The course identifier
+        type: string
+      batchId:
+        description: The batch identifier
+        type: string
+      template:
+        $ref: '#/definitions/AddBatchTemplate'
+    required:
+    - courseId
+    - batchId
+    - template
+  AddBatchTemplate:
+    title: AddBatchTemplate
+    type: object
+    properties:
+      identifier:
+        description: The template identifier
+        type: string
+      criteria:
+        description: The filter for the template
+        type: object
+        properties:
+          enrollment:
+            type: object
+            properties:
+              status: 
+                type: string
+                example: '2'
+      signatoryList:
+        description: The list of signatories 
+        type: array
+        items:
+          type: object
+          properties:
+            name:
+              description: The signatory name
+              type: string
+            id:
+              description: The identifier for the signatory
+              type: string
+            designation:
+              description: The designation of the signatory
+              type: string
+              example: CEO
+            image:
+              description: Signatory id
```

Comment:
@rahul-tarento @maheshkumargangula is this description the image of the signatories signature? please check

Reply:
corrected